nx/include/switch/services/news.h

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file news.h
+ * @brief News service IPC wrapper.
+ * @author Behemoth
+ * @copyright libnx Authors
+ */
+
+#pragma once
+
+#include "../kernel/event.h"
+#include "../services/acc.h"
+#include "../sf/service.h"
+#include "../types.h"
+
+typedef enum {
+    NewsServiceType_Application,
+    NewsServiceType_Starter,
+    NewsServiceType_Menu,
+    NewsServiceType_DataErase,
+    NewsServiceType_v,
```

Comment:
Do we know what the v actually stands for?

Reply:
Unused in official firmware npdms.

Reply:
Added a comment elsewhere with the real names, please use them.